Makefile

```diff
@@ -69,7 +69,7 @@ integration-only: install-tools ## Run integration tests
 		version \
 		generation \
 		team_members \
-		vulnerability"
+		vulnerability" -run=$(regex)
```

Comment:
This is more a Go functionality rather than Makefile. From their [docs](https://pkg.go.dev/cmd/go#hdr-Testing_flags):
```
-run=""
	if non-empty, specifies a regular expression to select
	directives whose full original source text matches the
	expression.
```

So yeah, it is optional. When not specified it runs all tests.

Reply:
<img src="https://media4.giphy.com/media/RGRkDWsTGvdUaQTO0z/giphy.gif"/>